### ISSUE TITLE
better instruction when user is offline

### ISSUE THREAD

**Issue by atwayne** [a85/POSTMan-Chrome-Extension#9812]
**Is your feature request related to a problem? Please describe.**

User is frustrated when 

1. User tries to use postman v8 without internet for the first time
2. and for some reason user has no awareness of the Scratch Mode at all

The error messages are not helpful at all. Here are all the messages I can find and none of them is about the scratch mode.

- Unable to load workspaces as you're offline
- You can switch to workspaces once you're back online
- Unable to load data as you're offline
- Your data may not be up to date until you're back online

What make things worse is that becase of the offline status, user can't search and ask questions on internet.

**Describe the solution you'd like**

Other than the current error messages, add a new instruction to the Scratch Mode like "Switch to Scratch Mode to use Postman offline".

Maybe a button to actually switch to scratch mode would be even better.

**Describe alternatives you've considered**
In a perfect solution, user should be able to access my workspace, may be a local cache, even user is offline. Make it readonly or discard the changes user made when offline is also a better option than not usable at all.
This could be another feature request but let's focus on better instruction on this one.

**Comment by DannyDainton** [comment]
Hey @atwayne

Can you confirm which app version you're using, please?

If you could also provide some screenshots of what you see what the app goes offline that would be really helpful.

There _should_ be a banner with the 'switch to scratchpad' link in it, across the top of the app - we'll need to check up on that if it's not coming through.

**Comment by atwayne** [comment]
I am working with v8.1.0 which I guess it's up to date. And this is a screenshot I took.

![image](https://user-images.githubusercontent.com/5857710/113991020-a10cae80-9884-11eb-905b-b27b6029b64e.png)

**Comment by atwayne** [comment]
The screenshot above was taken on the workspace tab and here is one for the Home tab
![image](https://user-images.githubusercontent.com/5857710/113991658-40ca3c80-9885-11eb-9615-a4b864fade7b.png)

**Comment by DannyDainton** [comment]
Looks like you have a pending update or it might be shown as failed, given that you're offline. 🤔 

Is the banner shown when you fully quit and open the app again? 

Just trying to fully establish what's happening here rather than only telling you to "switch it off and on again" 😄

**Comment by atwayne** [comment]
The update check failed due to offline. That's fine I guess.

The screenshots were actually taken after I came back online, restarted postman and confirmed that there is no pending update, switch to offline again, restarted postman and then take the screenshots.

And I have never see that banner during the whole process.

**Comment by DannyDainton** [comment]
Interesting - I'm on a Mac but that shouldn't impact the banner showing. 

You should be seeing this appearing at the top, with a link to where you can switch to the scratchpad.

With a Team user

![image](https://user-images.githubusercontent.com/17089546/113994376-3e4bf080-984d-11eb-8fba-2378b710eada.png)

And the same with a non-team user

![image](https://user-images.githubusercontent.com/17089546/113995121-de097e80-984d-11eb-9d62-ed69c61265dd.png)

**Comment by atwayne** [comment]
Yeah I am running a Windows version. Let's wait and see if it can be reproduced on someone else's setup.

**Comment by homestar9** [comment]
@atwayne Windows 10 user here and I have the exact same experience as you.  I like your perfect solution idea and hope Postman implements something like what you suggest.  It doesn't help us if we are trying to test something locally (and don't have internet) and we don't have access to a cached version of our workspace.

_"In a perfect solution, user should be able to access my workspace, may be a local cache, even user is offline. Make it readonly or discard the changes user made when offline is also a better option than not usable at all.
This could be another feature request but let's focus on better instruction on this one."_

**Comment by DannyDainton** [comment]
Just to clarify @homestar9 - You have the exact same experience of not seeing the banner, pointing to where you can find the Scratch Pad option when offline?

Are you able to see and use the Scratch Pad for making those local calls while offline?

**Comment by homestar9** [comment]
Correct. Here's a screenshot of what I see. I wish there was a way to view a cached version of the workspace so I could work offline.  From what I know about scratch pad (I haven't used it much), it does not give me access to a cached version of my workspace.

![image](https://user-images.githubusercontent.com/5341142/114081597-9ab90b00-9861-11eb-9abf-2bb71d12d4ed.png)

Same version as OP: v8.1.0

**Comment by myckhel** [comment]
@DannyDainton Please just bring back the offline mode from v7 🙏🏽  and don't make internet required again.

v8 has made life difficult for us so far without internet.

scratch pad is no where near usefulness 😭 😢 😭
